src/analysis/hands.cc

```diff
@@ -309,8 +313,7 @@ bool isOpenPinfu(const GameState& state, int player,
 
 bool isComplete(const GameState& state, int player) {
   auto root = breakdownHand(state.hands.at(player).live);
-  if (!root->IsComplete() && (isThirteenOrphans(state, player) == 0) &&
-      (isSevenPairs(state, player) == 0)) {
+  if (!root->IsComplete() && !isThirteenOrphans(state, player)) {
```

Comment:
so is no seven pairs check required in isComplete anymore?

Reply:
I removed the restriction from handtree that only allowed breakdowns to contain a single pair. So a seven pair hand should be properly broken down and root->IsComplete will return true for it.